src/lando/tests/test_version.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import subprocess
+
+
+def test_version():
+    # explicitly generate the version.py file since we can't
+    # guarantee it already exists in the testing environment
```

Comment:
Please use formal English in your comments (ie proper punctuation, capitalization, etc).

```suggestion
    # Explicitly generate the version.py file since we can't
    # guarantee it already exists in the testing environment.
```